next/components/ui/Videos/Videos.tsx

```diff
@@ -1,30 +1,25 @@
-// @ts-strict-ignore
-import { HorizontalScrollWrapper } from '@bratislava/ui-bratislava'
+import { ChevronRight } from '@assets/images'
+import { ComponentSectionsVideos } from '@bratislava/strapi-sdk-homepage'
+import { Button, HorizontalScrollWrapper } from '@bratislava/ui-bratislava'
+import { shouldShowButtonContent, VideoAttribute } from '@bratislava/ui-bratislava/Videos/VideosService'
 import cx from 'classnames'
 import React from 'react'
 
-interface IVideo {
-  title?: string
-  speaker?: string
-  url?: string
-  size?: 'default' | 'small'
-}
-
-export interface VideosProps {
-  id?: string
+export interface VideosProps extends ComponentSectionsVideos {
+  id: string
   className?: string
   title?: string
   subtitle?: string
-  videos?: IVideo[]
+  videos?: VideoAttribute[] | null
```

Comment:
If `videos` is mandatory prop but still can be undefined, we prefer to write `videos: VideoAttribute[] | null | undefined`.